### DISCUSSION TITLE
no logs in kibana

### DISCUSSION THREAD

**Discussion by ktroberson** [Security-Onion-Solutions/securityonion#10777] [Unsupported Versions]
When I run so-elasticsearch-indicies-list, I dont see any of the indicies, but when i log into the search nodes I do. Please help

**Comment by ktroberson** [comment]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/29292838/a5b41a68-9c91-47f2-a19f-ac9240af65b9)

**Reply by ktroberson** [reply]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/29292838/02481c21-880a-4fbe-8864-04706238d5b3)

**Comment by cm-ops** [comment]
`salt-call pillar.get elasticsearch:true_cluster` if that is `False`, you are running a cross cluster https://docs.securityonion.net/en/latest/elasticsearch.html#cross-cluster-search The indices are on the search nodes and the manager queries the search nodes via cross cluster search.
